packages/components/modal/src/components/osds-modal/osds-modal.scss

```diff
@@ -50,10 +50,12 @@
     }
 
     .content {
+      max-height: 30vh;
```

Comment:
is this a design prerequisite?

Reply:
Yes !
You can see it in comment in the task